app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,24 @@
  *= require font-awesome
  */
 
+
+.searchLocation {
+  height: 34px;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #555;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.